config/config.go

```diff
@@ -448,7 +448,14 @@ type CertificatesInformationIndicators struct {
 
 // Clustering defines configuration around multi-cluster functionality.
 type Clustering struct {
-	EnableExecProvider bool `yaml:"enable_exec_provider,omitempty" json:"enable_exec_provider"`
+	EnableExecProvider bool       `yaml:"enable_exec_provider,omitempty" json:"enable_exec_provider"`
+	KialiURLs          []KialiURL `yaml:"kiali_urls,omitempty" json:"kialiURLs,omitempty"`
```

Comment:
Should this be `kiali_urls` for the JSON tag as well? I can't seem to get these settings to apply when testing.